controllers/__init__.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+import os
+import re
```

Comment:
There's some linter errors that didn't get show up on yala, @italovalcy could you please add both these packages `controllers`, `db`, `models` [on this line](https://github.com/kytos-ng/mef_eline/blob/feat/persist-with-mongo/setup.py#L134)?